libraries/lib-sentry-reporting/SentryReport.h

```diff
@@ -0,0 +1,95 @@
+/*!********************************************************************
+
+ Audacity: A Digital Audio Editor
+
+ @file SentryReport.h
+ @brief Declare a class to report errors to Sentry.
+
+ Dmitry Vedenko
+ **********************************************************************/
+
+#pragma once
+
+#include <string>
+#include <utility>
+#include <vector>
+#include <memory>
+#include <functional>
+
+namespace audacity
+{
+namespace sentry
+{
+
+//! Additional payload to the exception
+using ExceptionData = std::pair<std::string, std::string>;
+
+//! A DTO for the Sentry Exception interface
+struct SENTRY_REPORTING_API Exception final
+{
+   //! Exception type. Should not have spaces.
+   std::string Type;
+   //! Message, associated with the Exception
+   std::string Value;
+   //! Arbitrary payload
+   std::vector<ExceptionData> Data;
+
+   bool Handled { false };
+   //! Create a new exception
+   static Exception Create(std::string type, std::string value);
+   //! Create a new exception with type *runtime_error*
```

Comment:
Do you mean "runtime_error"

I don't see the *'s in the code

Reply:
I really thought that Doxygen supports Markdown.

Reply:
You are right, it appears as italics in the html pages.